Lib/base64.py

```diff
@@ -241,29 +241,21 @@ def b32decode(s, casefold=False, map01=None):
     return bytes(decoded)
 
 
-# RFC 3548, Base 16 Alphabet specifies uppercase, but hexlify() returns
-# lowercase.  The RFC also recommends against accepting input case
-# insensitively.
 def b16encode(s):
     """Encode the bytes-like object s using Base16 and return a bytes object.
     """
     return binascii.hexlify(s).upper()
 
 
-def b16decode(s, casefold=False):
+def b16decode(s):
     """Decode the Base16 encoded bytes-like object or ASCII string s.
-
-    Optional casefold is a flag specifying whether a lowercase alphabet is
-    acceptable as input.  For security purposes, the default is False.
-
+    
     The result is returned as a bytes object.  A binascii.Error is raised if
```

Comment:
Please, could you remove one white-space on `object.  A binascii.Error` A little improve :-)

Reply:
Done :)